mezzanine/core/fields.py

```diff
@@ -94,7 +94,7 @@ def validate(self, value, instance):
             raise ValidationError(error)
 
     def value_to_string(self, obj):
-        value = self._get_val_from_obj(obj)
+        value = self.value_to_string(obj)
```

Comment:
Typo? Wasn't it supposed to be `value_from_object`?

Reply:
It was suppose to be value_from_object. The change made now causes an infinite recursion when serializing the field (for example, with dumpdata).

Reply:
Made a PR for this #1999